low-level/framework/base/sspl_constants.py

```diff
@@ -164,6 +164,33 @@
 
 SSPL_CONFIGS = ['log_level', 'cli_type', 'sspl_log_file_path', 'cluster_id', 'storage_enclosure', 'setup', 'operating_system']
 
+MODULES = {"CPUFAULTSENSOR": {"probe" : "sysfs", "threaded": "true"},
```

Comment:
@shubhamseagate Why we are moving configs to constants file here again? If you run through the ticket description, we need configurable parts in sspl.conf of LDR R2.

>Move all SSPL configs except common config to /etc/sspl.conf.
SSPL specific config is pretty much static but still remains with Provisioner in sls file and for any changes to this static sspl config requires provisioner involvement, which is unnecessary for Provisioner and delaying for SSPL team. So move back sspl static config back in sspl.conf. Also remove any unused kv pairs from sspl.sls or config fulfilled by common config already, so to avoid confusion.